src/ResourceManager/Sql/Commands.Sql/help/Set-AzureRmSqlDatabaseAuditing.md

```diff
@@ -57,29 +58,39 @@ PS C:\>Set-AzureRmSqlDatabaseAuditing -State Disabled -ResourceGroupName "Resour
 PS C:\>Set-AzureRmSqlDatabaseAuditing -State Enabled -ResourceGroupName "ResourceGroup01" -ServerName "Server01" -StorageAccountName "Storage22" -StorageAccountSubscriptionId "7fe3301d-31d3-4668-af5e-211a890ba6e3"
 ```
 
+### Example 4: Enable the extended auditing policy of an Azure SQL database
+```
+PS C:\>Set-AzureRmSqlDatabaseAuditing -State Enabled -ResourceGroupName "ResourceGroup01" -ServerName "Server01" -StorageAccountName "Storage22" -DatabaseName "Database01" -PredicateExpression "statement <> 'select 1'"
+```
+
+### Example 5: Remove the extended auditing policy of an Azure SQL database, and set an auditing policy instead of it.
+```
+PS C:\>Set-AzureRmSqlDatabaseAuditing -State Enabled -ResourceGroupName "ResourceGroup01" -ServerName "Server01" -StorageAccountName "Storage22" -DatabaseName "Database01" -PredicateExpression ""
+```
+
 ## PARAMETERS
 
 ### -AuditAction
 The set of audit actions.
 The supported actions to audit are:
 SELECT
-UPDATE
-INSERT
-DELETE
-EXECUTE
-RECEIVE
-REFERENCES
-The general form for defining an action to be audited is:
+UPDATE 
+INSERT 
+DELETE 
+EXECUTE 
+RECEIVE 
+REFERENCES 
+The general form for defining an action to be audited is: 
 [action] ON [object] BY [principal]
-Note that [object] in the above format can refer to an object like a table, view, or stored procedure, or an entire database or schema. For the latter cases, the forms DATABASE::[dbname] and SCHEMA::[schemaname] are used, respectively.
-For example:
+Note that [object] in the above format can refer to an object like a table, view, or stored procedure, or an entire database or schema. For the latter cases, the forms DATABASE::[dbname] and SCHEMA::[schemaname] are used, respectively. 
+For example: 
 SELECT on dbo.myTable by public
-SELECT on DATABASE::myDatabase by public
-SELECT on SCHEMA::mySchema by public
+SELECT on DATABASE::myDatabase by public 
+SELECT on SCHEMA::mySchema by public 
 For more information, see https://docs.microsoft.com/en-us/sql/relational-databases/security/auditing/sql-server-audit-action-groups-and-actions#database-level-audit-actions.
 
 ```yaml
-Type: System.String[]
+Type: String[]
```

Comment:
You've regenerated your help files incorrectly. You need to use the `-UseFullTypeName` switch as indicated in our documentation: https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md#updating-help-after-making-cmdlet-changes